packages/mobile/src/send/utils.ts

```diff
@@ -289,6 +289,14 @@ export function* handleSendPaymentData(
       isFromScan,
       origin: SendOrigin.AppSendFlow,
     })
+  } else if (isOutgoingPaymentRequest) {
+    navigate(Screens.SendAmount, {
```

Comment:
This is the exact same line as the `else` below, right? I think if we want to do this we can add `if (!canSendTokens && ! isOutgoingPaymentRequest) { throw error } to the condition below

Reply:
I had that in a pervious commit and wasn't sure if handling it in a new else if would, be more explicit. Happy to go back to what was in https://github.com/valora-inc/wallet/pull/1957/commits/f0f7228b1a5e92a713cd4645c961a66316722360.